acceptance_tests/features/environment.py

```diff
@@ -16,27 +24,32 @@ def before_all(context):
     database_controller.execute_rm_sql('resources/database/database_reset_rm.sql')
     database_controller.reset_ras_database()
     common.signed_in_rops(context)
+    execute_collection_exercises()
     enrolment_setup()
     common.signed_out_internal(context)
 
 
-def enrolment_setup():
+def execute_collection_exercises():
     test_file = 'resources/sample_files/business-survey-sample-date.csv'
-    sample_controller.load_sample(test_file=test_file)
-    collection_exercise_controller.execute_collection_exercise()
+    sample_controller.load_sample('bricks', '201801', test_file)
+    sample_controller.load_sample('bricks', '201812', test_file)
+    collection_exercise_controller.execute_collection_exercise(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54',
+                                                               period='201801')
+    collection_exercise_controller.execute_collection_exercise(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54',
+                                                               period='201812')
+    logger.info('Waiting for collection exercises execution process to finish...')
+    time.sleep(240)
```

Comment:
Hmm we really, really want to avoid sleeps in the code. Is there and API we can poll at the least to check for collection exercises

Reply:
I agree, there's a large (and inconsistent) period of time between the execution of the collection exercise and the generation of iac codes.
We could maybe make a call to some service to check if these have been created, I'll look into it

Reply:
After some conversations with @benjefferies it looks like there isn't an easy fix to this currently. I'm adding an issue on the repo + a tech debt card. It definitely isn't something that should be left for long